### PR TITLE
feat: requested amount listing view

### PR DESCRIPTION
**Pull request by sahas-01**

### CODE REVIEW

File: src/components/GovernanceCard.tsx [hunk 1]
```diff
@@ -16,9 +16,12 @@ import StatusTag from 'src/ui-components/StatusTag';
 import getRelativeCreatedAt from 'src/util/getRelativeCreatedAt';
 import { WarningMessageIcon } from '~src/ui-components/CustomIcons';
 import TopicTag from '~src/ui-components/TopicTag';
+import BigNumber from 'bignumber.js';
```
Comment (alphainfinitus) [inline src/components/GovernanceCard.tsx:19]: use bn.js instead

File: src/components/GovernanceCard.tsx [hunk 2]
```diff
@@ -84,34 +89,41 @@ const GovernanceCard: FC<IGovernanceProps> = (props) => {
 	const relativeCreatedAt = getRelativeCreatedAt(created_at);
 	const [tagsModal, setTagsModal] = useState<boolean>(false);
 
+	const tokenDecimals = chainProperties[network]?.tokenDecimals;
+	const requestedAmountFormatted = requestedAmount ? new BigNumber(requestedAmount).div(10 ** tokenDecimals).toFixed(0, BigNumber.ROUND_DOWN) : 0;
+	console.log('requestedAmountFormatted', requestedAmountFormatted);
```
Comment (alphainfinitus) [inline src/components/GovernanceCard.tsx:94]: lingering console log

File: src/components/GovernanceCard.tsx (hunk 1 above)
Comment (sahas-01) [reply src/components/GovernanceCard.tsx]: Resolved, pls check

File: src/components/GovernanceCard.tsx (hunk 2 above)
Comment (sahas-01) [reply src/components/GovernanceCard.tsx]: Removed, pls check

**Review by alphainfinitus — APPROVED** [review]

**Pull request merged**